install/install-base.php

```diff
@@ -131,4 +134,41 @@ public function new_site( $new_site ) {
 		$this->_activate();
 		restore_current_blog();
 	}
+
+	/**
+	 * Returns the plugin's options.
+	 *
+	 * @since 3.7
+	 *
+	 * @return Options
+	 */
+	protected function get_options(): Options {
+		if ( isset( $GLOBALS['polylang'], $GLOBALS['polylang']->model, $GLOBALS['polylang']->model->options ) ) {
```

Comment:
```suggestion
		if ( isset( $GLOBALS['polylang']->model->options ) ) {
```
It's sufficient IMHO. See https://onlinephp.io/c/8133d